ob_v3p0/common_credentials.lines

```diff
@@ -68,21 +89,26 @@ Package Credentials DataModel
         Mixin Extensions
 
     Include GeoCoordinates                                          // From CDM
+        Property type IRI 1                                         "The value of the type property MUST be an unordered set. One of the items MUST be the IRI 'GeoCoordinates'."
     
     Include IdentifierEntry                                         // From CDM
+        Property type IRI 1                                         "The value of the type property MUST be an unordered set. One of the items MUST be the IRI 'IdentifierEntry'."
 
     Class IdentityObject Unordered false []                         "A collection of information about the recipient of an achievement."
-        Property identityHash String 1                              "Either the IdentityHash of the identity or the plaintext value. If it's possible that the plaintext transmission and storage of the identity value would leak personally identifiable information where there is an expectation of privacy, it is strongly recommended that an IdentityHash be used."
-        Property type IRI 1                                         "The IRI of the property by which the recipient of the achievement is identified. This value should be an IRI mapped in the present context. For example, 'email' maps to 'http://schema.org/email' and indicates that the identity of the IdentityObject will represent a value of a Profile's 'email' property. See [[#profile-identifier-properties]]."
+        Property type IRI 1                                         "The value of the type property MUST be an unordered set. One of the items MUST be the IRI 'IdentityObject'."
         Property hashed Boolean 1                                   "Whether or not the `identity` value is hashed."
+        Property identityHash String 1                              "Either the IdentityHash of the identity or the plaintext value. If it's possible that the plaintext transmission and storage of the identity value would leak personally identifiable information where there is an expectation of privacy, it is strongly recommended that an IdentityHash be used."
+        Property identityType IRI 1                                 "The IRI of the property by which the recipient of the achievement is identified. This value should be an IRI mapped in the present context. For example, 'email' maps to 'http://schema.org/email' and indicates that the identity of the IdentityObject will represent a value of a Profile's 'email' property. See [[#profile-identifier-properties]]."
```

Comment:
Renamed `type` to `identityType` so `type` could be used for `@type`. Alternative solution: define all allowed IRIs in the context file as variations of IdentityObject:
```
    "email": {
      "@id": "https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#IdentityObject",
      "@context": {
        "hashed": {
          "@id": "https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#hashed", 
          "@type": "xsd:boolean"
        },
        "identityHash": {
          "@id": "https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#identityHash", 
          "@type": "xsd:string"
        },
        "identityType": {
          "@id": "https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#identityType", 
          "@type": "xsd:string"
        },
        "salt":  {
          "@id": "https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#salt", 
          "@type": "xsd:string"
        }
      }
    }
```